tests/RobotFramework/tests/cumulocity/log/log_operation.robot

```diff
@@ -30,13 +30,26 @@ Request with non-existing log type
     ...    failure_reason=.*No such file or directory for log type: example1
     ...    timeout=120
 
+Manual log operation request
```

Comment:
We also need a similar test for config_update and config_snapshot.

Reply:
I'll these system tests

Reply:
Systems tests have been provided in 673135cac7ce86cecc4380699234141ead537675 to cover more commands.